tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestDateOrder.java

```diff
@@ -403,10 +409,15 @@ public void testIso8601() {
 
         String[][] tests = {
             // only the first 6 elements of the path matter
+            {
+                "//ldml/dates/calendars/calendar[@type=\"iso8601\"]/dateFormats/dateFormatLength",
+                "M yy",
+                "Field yy is incorrect. For a YMD calendar, the year field cannot be truncated to 2 digits."
+            },
             {
                 "//ldml/dates/calendars/calendar[@type=\"iso8601\"]/dateFormats/dateFormatLength",
                 "M y",
```

Comment:
Sample test message: the path, the value, and then an error message (or null, if there is no problem)